src/kratos.ts

```diff
@@ -176,6 +177,10 @@ export const logout = async (
       req.log.warn({ err: e }, 'unexpected error while logging out');
     }
   }
-  await clearAuthentication(req, res, 'manual logout');
+  await clearAuthentication(
+    req,
+    res,
+    isDeleted ? 'deletion logout' : 'manual logout',
```

Comment:
It can actually still be split up more as from the frontend this can get triggered due to 2 reasons.
Maybe we want to enable that level of detail?

Reply:
Doesn't hurt. How do we tell them apart? Maybe add something in the frontend that differentiate the calls?

Reply:
Maybe also better tracked on the frontend since we have all the info there, api can stay as global counter.

Reply:
Yep we'd need to send something from fe to the logout call basically.
Can differentiate between actual manual logout and existing_session_logout or something

Not super urgent though

Reply:
@capJavert Frontend already tracks these as events. (there are cases where api will do logic though, that's why it's in 2 places now)

Reply:
Not sure what all the different cases are as well.

Reply:
@rebelchris nice, from my point of view if frontend tracka it no need to also do it here.

Reply:
Github is slow with showing comments to day... So we have these to cases that Ole-Martin added and two are tracked on the frontend?

Reply:
Send this error ID as an argument inside `logout`
https://github.com/dailydotdev/apps/blob/5c716e2c6669e994dbac12e197a47f3e51b0cc57/packages/shared/src/hooks/useRegistration.ts#L85-L90

And inside the `logout` function, we add that argument as a `?reason=`
https://github.com/dailydotdev/apps/blob/5c716e2c6669e994dbac12e197a47f3e51b0cc57/packages/shared/src/lib/user.ts#L99-L104

which we then override here?

Reply:
Only thing that comes to mind is that it would be possible to fake it on frontend and send `reason=whatever` but not blocking logic if it helps us track this better.

Reply:
Added reasoning on frontend https://github.com/dailydotdev/apps/pull/2805